packages/devtools_app/lib/src/memory/memory_controller.dart

```diff
@@ -2,111 +2,23 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
```

Comment:
add a comment that this file is now deprecated and should be removed when we switch to the package:flutter version of the app.

Reply:
Done in both memory_controller and memory_protocol.